src/components/UserList/UserListToolbar.tsx

```diff
@@ -34,7 +34,7 @@ export function UserListToolbar(props: UserListToolbarProps) {
         <>
           {selected.every((row) => {
             /*back doesn't have a pending type yet, this will need to be changed later*/
-            return row.roles?.at(0)?.id === UserTypesId.PENDING;
+            return row.roles?.at(0) === undefined;
```

Comment:
I think if user is in status `PENDING` then roles array is just empty. At least this is what backend team said last time